### DISCUSSION TITLE
Access for other App Stores?

### DISCUSSION THREAD

**Discussion by btucker22** [abentele/Erbele#58] [General]
Anyway we can download it another country App Store?  Happy to pay and download - just trying to work around country restrictions in the App Store.

**Comment by abentele** [comment]
I'm sorry to tell you that Erbele currently isn't available in the App Store any more due to expired certificate. Due to the fact that I would need a yearly subscription for the Apple Developer ID which costs ~$100 per year, and my income with this app (I don't have other apps in the app store) is even lower, I've decided to cancel the subscription.

As I down-prioritized my activities for Erbele, I'm not sure when and if at all I will re-activate it in app store again. You can still download the sourcecode and try to compile it with Xcode by yourself.

**Comment by btucker22** [comment]
No worries at all. I totally understand!

On Apr 24, 2024, at 3:46 PM, Andreas Bentele ***@***.***> wrote:

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>


I'm sorry to tell you that Erbele currently isn't available in the App Store any more due to expired certificate. Due to the fact that I would need a yearly subscription for the Apple Developer ID which costs ~$100 per year, and my income with this app (I don't have other apps in the app store) is even lower, I've decided to cancel the subscription.

As I down-prioritized my activities for Erbele, I'm not sure when and if at all I will re-activate it in app store again. You can still download the sourcecode and try to compile it with Xcode by yourself.
